vermouth/forcefield.py

```diff
@@ -57,9 +57,6 @@ def __init__(self, directory=None, name=None):
             self.name = os.path.basename(directory)
         if name is not None:
             self.name = name
-        if directory is not None and name is not None:
-            msg = 'Only one of name or directory should be provided. Not both.'
-            raise TypeError(msg)
```

Comment:
Why not both? Which takes priority?

Reply:
It use to prevent using both; by removing this test, "name" takes priority. I need to update the documentation.